examples/wagemos-cosmoskit-nextjs/src/app/remote/page.tsx

```diff
@@ -0,0 +1,131 @@
+'use client'
+import { stringToAccountId } from '@abstract-money/core'
+import {
+  useRemoteAccountIds,
+  useRemoteHosts,
+  useSignAndBroadcast,
+} from '@abstract-money/react'
+import { useAccounts } from '@abstract-money/react'
+import { useCreateRemoteAccount } from '@abstract-money/react'
+import { useConfig } from '@abstract-money/react'
+import { useExecuteOnRemote } from '@abstract-money/react'
+import { useChain } from '@cosmos-kit/react'
+import { useCallback, useMemo, useState } from 'react'
+import { Button } from '../../components/ui/button'
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../../components/ui/select'
+import { WalletButton } from '../_components/wallet-button'
+
+const CHAIN_NAME = 'neutron'
+export default function RemotePage() {
+  const { data: remoteHosts, isLoading, refetch } = useRemoteHosts(CHAIN_NAME)
+  const [chainInput, setChainInput] = useState(remoteHosts?.[0]?.[0])
+
+  const { address } = useChain(CHAIN_NAME)
+
+  const { data: accounts, status } = useAccounts(
+    {
+      args: {
+        chainName: CHAIN_NAME,
+        owner: address || '',
+      },
+    },
+    {
+      enabled: !!address,
+    },
```

Comment:
You should pass an `undefined` in `args` if some of such are not defined. By that you would not need to pass `enabled` option explicitly.

```suggestion
      args: address ? {
        chainName: CHAIN_NAME,
        owner: address,
      }: undefined,
    },
```